auv_setup/config/robots/beluga.yaml

```diff
@@ -17,8 +17,8 @@
 physical:
   weight: 24.00     # kg
   buoyancy: 23.00   # kg
-  center_of_mass: [-15.2, 1.0, 9.4]        # mm (x,y,z)
-  center_of_buoyancy: [-15.1, 0.9, 20.0]   # mm (x,y,z)
+  center_of_mass: [0.0, 0.0, 0.0]        # mm (x,y,z)
+  center_of_buoyancy: [0.0, 0.0, 0.0]   # mm (x,y,z)
```

Comment:
Should be updated once hardware is entirely done with their work. Do we have an overview of where these values are used? I believe they are used in the thrust allocator, but potentially elsewhere as well.